release/src/main/scripts/run_rc_validation.sh

```diff
@@ -285,6 +285,17 @@ if [[ ("$java_mobile_game_direct" = true || "$java_mobile_game_dataflow" = true)
     -PbqDataset=${MOBILE_GAME_DATASET} \
     -PpubsubTopic=${MOBILE_GAME_PUBSUB_TOPIC} \
     -PgcsBucket=${USER_GCS_BUCKET:5}  # skip 'gs://' prefix
+
+    echo "**************************************************************************"
+    echo "* Java mobile game on DataflowRunner using Beam GCP BOM: UserScore, HourlyTeamScore, Leaderboard"
```

Comment:
Should this be included as part of the release validation spreadsheet?
In other words, is this required to pass to validate the RC?